beacon_node/client/src/metrics.rs

```diff
@@ -15,6 +15,13 @@ pub static IS_SYNCED: LazyLock<Result<IntGauge>> = LazyLock::new(|| {
     )
 });
 
+pub static IS_OPTIMISTIC_SYNC: LazyLock<Result<IntGauge>> = LazyLock::new(|| {
+    try_create_int_gauge(
+        "optimistic_sync",
+        "Metric to check if the beacon chain is in optimistic sync mode. 0 if optimistic sync and 1 if synced",
```

Comment:
I think this is counterintuitive. In my original comment I was thinking we would name the metric something like `execution_verified 0|1`, with 0 representing the optimistic case and 1 representing full sync. _Or_ if we keep the `optimistic_sync` name I think we should swap it so that 1 = optimistic, 0 = not optimistic. That might actually be the clearest. We would just need to display the 0 as "good" in our grafana chart

Reply:
Yeah looks good! Thanks!